web/explorer/src/components/RuleMatchesTable.vue

```diff
@@ -0,0 +1,289 @@
+<template>
+    <div class="card">
+        <TreeTable
+            :value="filteredTreeData"
+            v-model:expandedKeys="expandedKeys"
+            size="small"
+            :filters="filters"
+            :filterMode="filterMode"
+            sortField="namespace"
+            :sortOrder="-1"
+            removableSort
+            :rowHover="true"
+            :indentation="1.3"
+            selectionMode="single"
+            @node-select="onNodeSelect"
+            :pt="{
+                row: ({ instance }) => ({
+                    oncontextmenu: (event) => onRightClick(event, instance)
+                })
+            }"
+        >
+            <template #header>
+                <div class="flex justify-content-end w-full">
+                    <IconField>
+                        <InputIcon class="pi pi-search" />
+                        <InputText v-model="filters['global']" placeholder="Global search" />
+                    </IconField>
+                </div>
+            </template>
+
+            <!-- Rule column (always visible) -->
```

Comment:
what's not always visible? can you clarify the comment please?

Reply:
Sorry for the confusion here, this was an old comment (now removed) for when we had toggeable columns and rule column used to always be visible. Now by default, all columns are visible.